README.md

```diff
@@ -13,105 +46,42 @@ See [below](#collaborative-and-publication-workflows) for special cases like pub
-  -m MANIFEST, --manifest MANIFEST
-                        Manifest file (default: cbio_file_name_id.txt from Step 1 output)
-  -f FILE_TYPES, --file-types FILE_TYPES
-                        Comma-separated file types to download
-  -at AWS_TBL, --aws-tbl AWS_TBL
-                        AWS table with bucket name and keys
-  -sp SBG_PROFILE, --sbg-profile SBG_PROFILE
-                        SBG profile name
-  -c CBIO, --cbio CBIO  cBio manifest to limit downloads
-  -ao, --active-only    Only include active files
-  -rm, --rm-na          Remove entries where file_id and s3_path are NA
-  -d, --debug           Enable debug mode
-  -o, --overwrite       Overwrite files if they already exist
-  -ms MANIFEST_SUBSET, --manifest-subset MANIFEST_SUBSET
-                        Check that files were downloaded. Default: manifest_subset.tsv from Step 3
-  -dc DATA_CONFIG, --data-config DATA_CONFIG
-                        Data processing configuration file. Default: value inputted for --study + '_data_processing_config.json'
-  -dgd {both,kf,dgd}, --dgd-status {both,kf,dgd}
-                        Flag to determine load will have pbta/kf + dgd(both), kf/pbta only(kf), dgd-only(dgd). Default: kf
-  -l, --legacy          Enable legacy mode
-```
 
-## Run tool
+### Usage
 ```sh
-cbioportal_etl \
-    --steps all \
+cbio-etl import \
     --db-ini /path/to/db.ini \
-    --token /path/to/cbioportal_data_access_token.txt \
-    --study oligo_nation \
-    --sbg-profile default
+    --study pbta_pnoc \
+    --sbg-profile default \
+    --dgd-status kf 
   ```
 
-### Required credentials files
-- Copy the `credentials_templates/template.db.ini` template to `/path/to/db.ini` and replace placeholders with your credentials.
-- Copy the `credentials_templates/template.sevenbridges.ini` template to `~/.sevenbridges/credentials` and replace placeholders with your credentials.
-- Download a reusable access token for PedcBioPortal `cbioportal_data_access_token.txt` from [here](https://pedcbioportal.kidsfirstdrc.org/webAPI#using-data-access-tokens).
-
-### Steps Argument
-The `--steps` argument specifies which steps of the pipeline to run. It is outlined as follows:
-1. Generate config JSON
-1. Get study metadata
-1. Compare current DWH data vs cBioPortal build
-1. Get files from manifest
-1. Check downloaded files
-1. Build genomic file package
-
-You can specify the steps in one of the following ways:
-- **Run a single step**:
-  ```bash
-  --steps 1
-  ```
-  This will only execute Step 1 (Get study metadata).
-- **Run multiple steps**:
-  ```bash
-  --steps 2 3 4
-  ```
-  This will execute Steps 2, 3, and 4 in sequence.
+## Docker Installation
+### Installation Steps
+```sh
+docker pull pgc-images.sbgenomics.com/d3b-bixu/cbio-etl:1.0.0
+```
 
-- **Run the whole ETL**:
-  ```bash
-  --steps all
-  ```
-  This will execute Steps 1 through 5.
+### Usage
+```
+docker run --rm -it \
+    -v /path/to/db.ini:/credentials/db.ini \
```

Comment:
Oh interesting, is this like mounting each file instead mounting an entire directory?